app/src/main/res/values/strings.xml

```diff
@@ -903,10 +903,10 @@
     <string name="private_browsing_exit_tooltip">Exit Private Browsing</string>
 
     <!-- This string is shown in the context menu after a longpress on a link. When clicked it opens the target url in a new window. -->
-    <string name="context_menu_open_new_window">Open in a new window.</string>
+    <string name="context_menu_open_new_window_1">Open in a new window</string>
 
     <!-- This string is shown in the context menu after a longpress on a link. When clicked it opens the target url in a new tab. -->
-    <string name="context_menu_open_new_tab">Open in a new tab.</string>
+    <string name="context_menu_open_new_tab">Open in a new tab</string>
```

Comment:
Looks like you changed the version on `context_menu_open_new_window` but not `context_menu_open_new_tab`

Reply:
context_menu_open_new_tab landed today, is not translated yet. Should we change that one too?

Reply:
I'm not sure how it works but I would change it just to be safe.

Reply:
done